scripts/tablegen/tablegen.py

```diff
@@ -75,11 +78,12 @@ def decode_helm_yaml(s):
     totalNum = len(context)
 
     for lineNum in range(0, totalNum):
+
         if  context[lineNum].strip().startswith('- '):
             pass
         elif '#' in context[lineNum] and '#' == context[lineNum].lstrip()[0]:
-            if "Description: " in context[lineNum]:
-                desc = context[lineNum].strip()
+            desc = context[lineNum].strip()
```

Comment:
Our original prototype for this work had this technique of generating desc from a tag.  We sort of decided it was unwieldy and might be prone to error, especially if developers didn't understand how to write the tag.  What would be preferred is parsing the comments character `#` that proceeds any K/V pair, joining it together, and removing the `#` and any extra whitespace.  Since we have to modify a whole lot of values.yaml (each of the subcharts has a `values.yaml` that needs parsing as well) this should yield a pretty quick docification of `values.yaml` throughout the codebase.